build/configs/common_download.sh

```diff
@@ -433,11 +433,18 @@ IFS=',' read -ra sizes <<< "$sizes"
 #Calculate Flash Offset
 num=${#sizes[@]}
 offsets[0]=`printf "0x%X" ${CONFIG_FLASH_START_ADDR}`
-
+flag=0
 for (( i=1; i<=$num-1; i++ ))
 do
 	val=$((sizes[$i-1] * 1024))
-	offsets[$i]=`printf "0x%X" $((offsets[$i-1] + ${val}))`
+	if [[ $val == 8192 ]];then
+		flag=1
+	fi
+	if [[ $flag == 1 ]];then
+		offsets[$i]=`printf "0x%X" $((offsets[$i-1] + ${val} + 16384 * 1024))`
```

Comment:
`16384` looks rtl8730e specific. How about `8192` in line 440? All are common for entire chipsets?

Reply:
this change has been removed